src/components/balance/Wallet.vue

```diff
@@ -18,16 +18,15 @@
         </div>
       </div>
 
-      <button type="button" class="icon tw-ml-auto tw-tooltip" @click="openModal">
+      <button type="button" class="icon tw-ml-auto tw-tooltip" @click="disconnectAccount">
```

Comment:
![Screen Shot 2022-01-04 at 11 17 43 AM](https://user-images.githubusercontent.com/40356749/148044279-990e384e-f230-41c5-b69b-c0b3192ff66d.png)

Currently, clicking the logout button will disconnect the account and show the login button.

![Screen Shot 2022-01-04 at 11 18 37 AM](https://user-images.githubusercontent.com/40356749/148044388-91db5c9e-8978-4f32-ba14-8fd085bce80e.png)
How about we display the wallet connection modal after disconnecting the user, so we reduce the number of clicks when the user wants to switch the account?

In other words, instead of click logout -> click login -> click wallet to connect, we just make it click logout -> (disconnect the account) -> automatically show the wallet connect modal

Reply:
Ok, fixed.
https://gyazo.com/7d9a4571b5112a09e2f09871c586201a

Reply:
I've added toast and disabled buttons for MetaMask account.
![image](https://user-images.githubusercontent.com/92044428/148102130-0518661e-2fe2-4708-aeba-0a3ee50d68af.png)

Reply:
And I have updated icons.

![image](https://user-images.githubusercontent.com/92044428/148102279-b58f9d7f-e1d3-406d-aa3f-6dbb688e3c3f.png)